### PR TITLE
chore: follow-up to abstract getTokenValueKey and getTokenTypeKey into utility functions

### PR DESCRIPTION
**Pull request by akshay-gupta7**
Follow up to PR #3024 

- Move getTokenValueKey and getTokenTypeKey into utility files

### CODE REVIEW

File: packages/tokens-studio-for-figma/src/utils/getTokenTypeKey.ts
Comment (cuserox) [inline packages/tokens-studio-for-figma/src/utils/getTokenTypeKey.ts:1]: Can we create tests to cover its basic functionality?

File: packages/tokens-studio-for-figma/src/utils/getTokenValueKey.ts
Comment (cuserox) [inline packages/tokens-studio-for-figma/src/utils/getTokenValueKey.ts:1]: Similar about writing tests for this 🙏

**Review by cuserox — CHANGES_REQUESTED** [review]
Some base unit tests would be nice 👼 

Also missing some linting!

File: packages/tokens-studio-for-figma/src/utils/getTokenValueKey.ts
Comment (akshay-gupta7) [reply packages/tokens-studio-for-figma/src/utils/getTokenValueKey.ts:1]: Added some tests

**Pull request merged**